script/results.html

```diff
@@ -0,0 +1,256 @@
+                // Go down the rabbit hole
+                for (const key in input) {
+                    text += '<tr><td><b>' + key + '</b></td><td>'
+                    text += innerFill(input[key]) + '</td></tr>'
+                }
+                text += '</tbody></table>'
+            } else {
+                // What else could it be?
+                text = input
+            }
+            return text
+        }
+
+
+        function fillConfig(input, element) {
+            $(element).empty()
+            var text ='<tbody>'
+            $(text).appendTo($(element))
+
+            for (const key in input) {
+                text = '<tr><td><b>' + key + '</b></td><td>'
+                // Go down the rabbit hole
+                text += innerFill(input[key]) + '</td></tr>'
+                $(text).appendTo($(element))            
+            }
+
+            text ='</tbody>'
+            $(text).appendTo($(element))
+        }
+
+        function fillResults(input, element) {
+            $(element).empty()
+            // The results object is composed of multiple objects, 1 per test
+            // Inside the object we have an array of results
+            id = 1
+            var ansi_up = new AnsiUp;
+
+            tests_total = 0
+            tests_passed = 0
+            tests_skipped = 0
+            tests_failed = 0
+
+            // Compute number of passed/skipped/failed results
+            for (const key in input) {
+                for (const item of input[key]) {
+                    if (item.state == 'passed') {
+                        tests_total++
+                        tests_passed++
+                    } else if (item.state == 'skipped') {
+                        tests_total++
+                        tests_skipped++
+                    } else if (item.state == 'failed') {
+                        tests_total++
+                        tests_failed++
+                    }
+                }
+            }
+
+            // First we will have a left column in the table with the summarized results
+            text ='<thead><tr><th scope="col">Test summary</th><th scope="col">Test results</th></tr></thead><tbody>'
+            text += '<tr><td class="align-top"><b>Total:</b> ' + tests_total + '<br><b>Passed:</b> ' + tests_passed + '<br><b>Skipped:</b> ' + tests_skipped + '<br><b>Failed:</b> ' + tests_failed + '</td><td>'
+            text += '<div class="accordion" id="results-accordion">'
+
+            for (const key in input) {
+                // NOTE: we are assuming the test result is determined by the passed/failed state of the first item
+                status = input[key][0].state
+                if (status === "passed") {
+                    buttontype = 'bg-success text-white'
+                } else if (status === "skipped") {
+                    buttontype = 'bg-warning text-black'
+                } else {
+                    buttonntype = 'bg-danger text-white'
```

Comment:
This doesn't seem to change the bg color, unlike bg-warning. See the screen shot I sent to you

Reply:
Double 'n' in buttontype, /me facepalms